### ISSUE TITLE
Can I use a custom SVG icon in my own admonitions?

### ISSUE THREAD

**Issue by danieljfarrell** [FlandiaYingman/note-me#5]
Hello,

Thanks for your helpful plugin, I'm using it a lot.

I had a question about using custom svg files? I was trying to make a custom admonition called "decision" that would use a check mark icon: <img src="https://github.com/user-attachments/assets/097bc3da-74bf-45ad-b648-545535a6ee34" width="25" height="25">

```typst
#admonition(
  icon: "check.svg",
  color:  rgb(16, 166, 74),
  title: "Decision",
)[Oh no the icon cannot be found!]
```

Here is a screenshot from typst the error says "File not found". I am assuming I am doing something stupid.

<img width="723" alt="custom_icon_file_example" src="https://github.com/user-attachments/assets/8a254c9c-2945-4611-93d3-93ac92d75504">

**Comment by flandia** [comment]
This is a bit problematic because Typst reads the file from the path relative to the package's source file. I think I can update the package in a few days to let it accept an SVG file content instead of SVG file path.

**Comment by flandia** [comment]
@danieljfarrell fixed in 0.3.0
